ruleset/rules/0690-gcp_rules.xml

```diff
@@ -283,6 +283,245 @@ ID: 65000 - 65499
         <options>no_full_log</options>
     </rule>
 
+    <!-- Mitre detection rules  -->
+    <!-- collection gcp_pub_sub subscription creation -->
+    <rule id="65051" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.pubsub\.v\d*\.Subscriber\.CreateSubscription$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP pub/sub topic subscription created</description>
```

Comment:
I'm not following what you want to detect here. What means topic subscription and how it's related to Data storage (T1530)?

Reply:
A pub/sub topic is where logs are sent, an undesired subscription may be made to a topic in order to access logs.

Reply:
Can be understanding as Storage as well?

Reply:
We both agree that the objective of an adversary subscribing to a topic is to access logs? The question would be if a topic is considered a data storage object. 

MITRE also mention the retreival of logs in order to gain acces to storage objects:
> Adversaries may also obtain leaked credentials in source repositories, logs, or other means as a way to gain access to cloud storage objects that have access permission controls.

So accessing the logs would be considered part of the same technique? What alternative technique do you propose if any?